src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseVersionsController.cs

```diff
@@ -67,27 +70,47 @@ public async Task<ActionResult<IdViewModel>> CreateReleaseAmendment(Guid release
                 .HandleFailuresOrOk();
         }
 
-        // We intend to change this route, to make these endpoints more consistent, as per EES-5895
-        [HttpGet("release/{releaseVersionId:guid}/data/{fileId:guid}")]
+        [HttpGet("releaseVersions/{releaseVersionId:guid}/data/{fileId:guid}")]
         public async Task<ActionResult<DataFileInfo>> GetDataFileInfo(Guid releaseVersionId, Guid fileId)
         {
             return await _releaseDataFileService
                 .GetInfo(releaseVersionId, fileId)
                 .HandleFailuresOrOk();
         }
 
-        // We intend to change this route, to make these endpoints more consistent, as per EES-5895
-        [HttpGet("release/{releaseVersionId:guid}/data")]
+        [HttpGet("releaseVersions/{releaseVersionId:guid}/data")]
         public async Task<ActionResult<List<DataFileInfo>>> GetDataFileInfo(Guid releaseVersionId)
         {
             return await _releaseDataFileService
                 .ListAll(releaseVersionId)
                 .HandleFailuresOrOk();
         }
 
+        [HttpGet("releaseVersions/{releaseVersionId:guid}/uploads")]
+        public async Task<ActionResult<List<DataSetUploadViewModel>>> GetDataSetUploads(
+            Guid releaseVersionId,
+            CancellationToken cancellationToken)
+        {
+            return await _dataSetUploadRepository
+                .ListAll(releaseVersionId, cancellationToken)
+                .HandleFailuresOrOk();
+        }
+
+        [HttpDelete("releaseVersions/{releaseVersionId:guid}/upload/{dataSetUploadId:guid}")]
+        public async Task<ActionResult<Unit>> DeleteDataSetUploads(
+            Guid releaseVersionId,
+            Guid dataSetUploadId,
+            CancellationToken cancellationToken)
+        {
+            return await _dataSetUploadRepository
+                .Delete(releaseVersionId, dataSetUploadId, cancellationToken)
+                .HandleFailuresOrOk();
```

Comment:
For a `DELETE` endpoint, it's probably better to just return a non-generic ActionResult and use `HandleFailuresOrNoContent()`.